indico/modules/rb_new/client/js/components/TimelineContent.jsx

```diff
@@ -152,7 +156,10 @@ function TimelineRowLabel({label, availability, longLabel}) {
     const roomLabel = (
         <span>
             {availability && <Icon name="circle" size="tiny" color={color} styleName="dot" />}
-            <span>{label}</span>
+            {onClickLabel
+                ? <a style={{cursor: 'pointer'}} onClick={onClickLabel}>{label}</a>
```

Comment:
Is this the only way to make it look like a link?

Reply:
I think the only other option would be adding a `href` attribute (which would then require you to preventDefault the regular click event)... or maybe we could set this cursor style on all `a` elements in rb-new? We don't use non-link anchors anyway.

Reply:
I got it from https://github.com/indico/indico/blob/master/indico/modules/rb_new/client/js/modules/bookRoom/BookRoomModal.jsx#L308 but I agree on setting this cursor style on all `a` elements in rb-new.